app-misc/gramps/gramps-4.2.4.ebuild

```diff
@@ -19,22 +19,24 @@ IUSE="+reports +exif spell"
 
 RDEPEND="
 	dev-python/bsddb3[${PYTHON_USEDEP}]
-	>=dev-python/pygobject-3.12:3[${PYTHON_USEDEP}]
+	dev-python/pycairo[${PYTHON_USEDEP}]
+	>=dev-python/pygobject-3.12:3[cairo,${PYTHON_USEDEP}]
 	dev-python/pyicu[${PYTHON_USEDEP}]
 	gnome-base/librsvg:2
-	x11-libs/cairo
-	>x11-libs/gtk+-3.14.8:3
+	>x11-libs/gtk+-3.14.8:3[introspection]
 	x11-libs/pango[introspection]
 	x11-misc/xdg-utils
 	reports? ( media-gfx/graphviz )
 	exif? ( >=media-libs/gexiv2-0.5[${PYTHON_USEDEP},introspection] )
-	spell? (
-		app-text/gtkspell[introspection]
-		dev-python/gtkspell-python
-	)
+	spell? ( app-text/gtkspell:3[introspection] )
 "
 
 PATCHES=(
 	"${FILESDIR}/${P}-resourcepath.patch"
-	"${FILESDIR}/${P}-versioned_doc_dir.patch"
 )
+
+src_prepare() {
```

Comment:
don't use `src_prepare` with distutils - use `python_prepare_all` and at the end call `distutils-r1_python_prepare_all`. Also, get rid of the `default`.